KosherCocoa/Library/Core/Calendar/Chagim/KCJewishCalendar.m

```diff
@@ -279,7 +289,7 @@ - (BOOL)isErevYomTov
     NSInteger holidayIndex = [self yomTovIndex];
 
     return holidayIndex == kErevPesach || holidayIndex == kErevShavuos || holidayIndex == kErevRoshHashana
-    || holidayIndex == kErevYomKippur || holidayIndex == kErevSuccos;
+    || holidayIndex == kErevYomKippur || holidayIndex == kErevSuccos || holidayIndex == kHoshanaRabba || (holidayIndex == kCholHamoedPesach && [self currentHebrewDayOfMonth] == 20);
```

Comment:
this is how it is in KosherJava

Reply:
Nice! I think this can go in its own PR. "Add Erev Yom Tov for Second Days of Sukkot and Pesach" or something like that.